llvm/lib/Target/X86/Disassembler/X86Disassembler.cpp

```diff
@@ -754,7 +783,7 @@ static int readModRM(struct InternalInstruction *insn) {
       return base + index;                                                     \
     case TYPE_R8:                                                              \
       index &= mask;                                                           \
-      if (index > 0xf)                                                         \
+      if (index > MAX_GPR_NUM)                                                 \
         *valid = 0;                                                            \
```

Comment:
I think we can remove the code now since there's no difference between fixupRegValue and fixupRMValue

Reply:
Not addressed

Reply:
You're right. Done.